homeassistant/components/generic/config_flow.py

```diff
@@ -356,6 +365,9 @@ async def async_step_init(
                     ],
                     CONF_FRAMERATE: user_input[CONF_FRAMERATE],
                     CONF_VERIFY_SSL: user_input[CONF_VERIFY_SSL],
+                    CONF_USE_WALLCLOCK_AS_TIMESTAMPS: user_input.get(
```

Comment:
I don't think `get` is needed if the new parameter is marked as 'required' in the schema.